supplier_app/models.py

```diff
@@ -57,6 +57,16 @@ def _token_generator(self):
         return hashlib.sha256(salt.encode('utf-8')).hexdigest()
 
 
+class EBEntity(models.Model):
+    eb_name = models.CharField(
```

Comment:
Should we consider the country for the EBEntity?

Reply:
Yes and also this will be useful for other entities